_sass/_layout.scss

```diff
@@ -81,6 +81,22 @@ article {
       padding: 0 0.5rem;
     }
   }
+
+  .thanks {
+    font-size: 0.8em;
+    font-style: italic;
+    text-align: center;
+
+    .links {
+        column-gap: 3em;
+        display: flex;
+        justify-content: center;
```

Comment:
Something doesn't seem to be working right with this in Epiphany stable or nightly, and so I'd assume Safari as well.

Reply:
![Screenshot from 2019-10-07 14-53-04](https://user-images.githubusercontent.com/611168/66347717-35095d00-e912-11e9-81fd-09634455eedd.png)

Reply:
Oh, it's probably the `column-gap` property. I'll have to change that to just margins